libraries/aws_db_subnet_groups.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDbSubnetGroups < AwsResourceBase
+  name 'aws_db_subnet_groups'
+  desc 'Verifies settings for a Subnet Groups in bulk'
+  example '
+    describe aws_db_subnet_groups do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table, :subnet_rows
+  FilterTable.create
+             .register_column(:db_subnet_group_names,                field: :db_subnet_group_name)
+             .register_column(:db_subnet_group_descriptions,         field: :db_subnet_group_description)
+             .register_column(:vpc_ids,                              field: :vpc_id)
+             .register_column(:subnet_group_status,                  field: :subnet_group_status)
+             .register_column(:subnets,                              field: :subnets)
+             .register_column(:db_subnet_group_arns,                 field: :db_subnet_group_arn)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    subnet_group_rows = []
+    pagination_options = {}
+    catch_aws_errors do
```

Comment:
Even though the API would return 100 subnet groups by default; I think it is safer to run this in a loop and look for a pagination token in case there are more.

Reply:
updated in recent commit